skt/executable.py

```diff
@@ -385,10 +387,15 @@ def setup_parser():
     """
     parser = argparse.ArgumentParser()
 
-    parser.add_argument("-d", "--workdir", type=str, help="Path to work dir")
+    parser.add_argument("-d", "--workdir", type=str,
+                        help="Path to work dir with kernel source git tree",
```

Comment:
I'm not sure we can write this kind of description for this option. Since workdir can also contain the build, and possibly other data. At least it should say "contains source as well as build, unless -b points somewhere else", or something like that.

Reply:
Just changed this back. Less relevant now that we no longer refer to SRCDIR